bootstrap/manifest_template.json

```diff
@@ -131,6 +131,34 @@
     "data": "{{ getenv \"CONTROLLER_KEY\" }}"
   },
   {
+    "id": "redis-api",
+    "app": {
+      "name": "redis-api",
+      "meta": {"flynn-system-app": "true"}
+    },
+    "action": "run-app",
+    "release": {
+      "env": {
+        "FLYNN_REDIS": "redis",
+        "CONTROLLER_KEY": "{{ (index .StepData \"controller-key\").Data }}",
+        "REDIS_IMAGE_URI": "$image_repository?name=flynn/redis&id=$image_id[redis]"
+      },
+      "processes": {
+        "web": {
+          "ports": [{"port": 80, "proto": "tcp"}],
+          "cmd": ["api"]
+        }
+      }
+    },
+    "artifact": {
+      "type": "docker",
+      "uri": "$image_repository?name=flynn/redis&id=$image_id[redis]"
+    },
+    "processes": {
+      "web": 2
+    }
+  },
+  {
```

Comment:
Yep, because the scheduler isn't fired up yet. Move down to where `blobstore` and friends are. The controller needs to be fully setup before `deploy-app` works.

I can probably test this today if my internet is fixed but it should work.